javascript/index.js

```diff
@@ -24,17 +24,113 @@
     document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
     document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
   }, (error) => console.log(error));
-
+ */
 
 
 // Iteration 1 - using callbacks
-// ...
+
+getInstruction(
+  "mashedPotatoes",0, 
+  (step1) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step1}</li>`;
+  getInstruction("mashedPotatoes", 1, 
+  (step2) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step2}</li>`;
+  getInstruction("mashedPotatoes", 2, 
+  (step3) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step3}</li>`;
+  getInstruction("mashedPotatoes", 3, 
+  (step4) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step4}</li>`;
+  getInstruction("mashedPotatoes", 4, 
+  (step5) => {
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>${step5}</li>`;
+  document.querySelector("#mashedPotatoesImg").removeAttribute("hidden");
+  document.querySelector("#mashedPotatoes").innerHTML += `<li>Mashed potatoes are ready!</li>`;
+  }, 
+  (error) => console.log(error));
+  }, 
+  (error) => console.log(error));
+  }, 
+  (error) => console.log(error));
+  }, 
+  (error) => console.log(error));
+  }, 
+  (error) => console.log(error));
 
 // Iteration 2 - using promises
```

Comment:
On this iteration, you should return the next function inside of the then:
```
obtainInstruction("steak", 0)
  .then((step0) => {
    document.querySelector("#steak").innerHTML += `<li>${step0}</li>`;
    return obtainInstruction("steak", 1);
  })
  .then((step1) => {
    document.querySelector("#steak").innerHTML += `<li>${step1}</li>`;
    obtainInstruction("steak", 2);
  })
  .catch((err) => console.log(err));
```
Until you had all the functions declared